types/apis/certmanager.k8s.io/v1alpha1/types.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+import (
+	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
+	"github.com/rancher/norman/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Certificate struct {
```

Comment:
Can we not just import this type?

Reply:
there are some errors from controller complaining about not finding the type so instead i created those crd first